internal/config/config.go

```diff
@@ -25,6 +35,8 @@ func setDefaultValuesForMainConfig() {
 	viper.SetDefault("hactar.api-url", "")
 	viper.SetDefault("log.level", "error")
 	viper.SetDefault("lotus.network-address", "t01")
+	viper.SetDefault("sentry.key", "")
```

Comment:
I don't see the usage of these configs properties?

Reply:
Good catch, forgot to delete that one after I've switched to env